### ISSUE TITLE
Combination of Sentence BERT and Naive Bayes raises error

### ISSUE THREAD

**Issue by taleevenhuis** [asreview/asreview#1480]
**Describe the bug**
When I combine in ASReview the Feature extractor **Sentence BERT** and the Classifier **Naive Bayes**, it reports the following error: _"ValueError: Negative values in data passed to MultinomiaINB (input X)"_

**To Reproduce**
Steps to reproduce the problem:

**Screenshots**
**_See also the screen shots of the command line in the attached file._** 
[SBERT_Naive Bayes.docx](https://github.com/asreview/asreview/files/11903421/SBERT_Naive.Bayes.docx)

**Version information**
 - OS: Windows
 - ASReview version 1.2

**Additional context**

**Comment by rohitgarud** [comment]
Hi @taleevenhuis, Naive Bayes is only compatible with TFIDF features. For using SBERT features, you have to use other classifiers.

**Comment by taleevenhuis** [comment]
Ah, thanks for your quick and useful reply, Rohit!

**Comment by Rensvandeschoot** [comment]
Indeed, the error message is not informative, but the combination should not be possible at all (just like some other combinations are ruled out)
